rubocop/Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'rspec'
+gem 'rubocop'
```

Comment:
I'm curious about this one @Goose97, wasn't it added in `Gemfile.tt` inside the `group :development, :test` scope? 🤔

Reply:
this Gemfile is for testing the custom rubocop, unlike the Gemfile.tt that will be created during the template bootstrap. In another word, the users of this template won't see this file, only us the developers 👍